### ISSUE TITLE
Send (Switch Name in addition to switch DPID) and (Table name in addition to Table ID) to CouchDB

### ISSUE THREAD

**Issue by shivarammysore** [REANNZ/faucet#398]
Today, when JSON object is received by CouchDB from Gauge, we get only Switch DPID.  Can we get the Switch name too?

Also, Faucet creates OF pipelines and pushes them into the switch.  When tables and flow info is sent to CouchDB, the JSON object contains Table numbers but not a friendly name.  Not having this means one has to look at the pipeline to derive the name.  Hence, request table name as programmed by the pipeline (VLAN_ACL, PORT_ACL, FLOOD, etc) be sent with TableID to CouchDB.
Another option here is that the table id-name mapping can be sent separately and stored in a different table which is refreshed if pipeline is changed.

In summary, both of the above features require name mapping for switch and table id be available in the Couch database so that applications have easy access to them for presentation to users.

**Comment by anarkiwi** [comment]
The CouchDB connector doesn't have tests, but, we now have Prometheus support, and most applications will probably want to scrape Prometheus also to get FAUCET internal state data.

FAUCET now exports the data you requested via Prometheus.

$ wget -q -O- localhost:9244|grep -e "^faucet_config"
faucet_config_table_names{dpid="0xeccd6df72de7",name="eth_dst"} 6.0
faucet_config_table_names{dpid="0xb827eb608918",name="ipv6_fib"} 5.0
faucet_config_table_names{dpid="0xeccd6df72de7",name="vlan_acl"} 2.0
faucet_config_table_names{dpid="0xb827eb608918",name="eth_src"} 3.0
faucet_config_table_names{dpid="0xb827eb608918",name="port_acl"} 0.0
faucet_config_table_names{dpid="0xeccd6df2df80",name="port_acl"} 0.0
faucet_config_table_names{dpid="0xeccd6df2df80",name="eth_dst"} 6.0
faucet_config_table_names{dpid="0xeccd6df2df80",name="eth_src"} 3.0
faucet_config_table_names{dpid="0xeccd6df72de7",name="ipv4_fib"} 4.0
faucet_config_table_names{dpid="0xeccd6df2df80",name="vlan_acl"} 2.0
faucet_config_table_names{dpid="0xeccd6df2df80",name="flood"} 7.0
faucet_config_table_names{dpid="0xeccd6df72de7",name="vlan"} 1.0
faucet_config_table_names{dpid="0xb827eb608918",name="vlan_acl"} 2.0
faucet_config_table_names{dpid="0xb827eb608918",name="eth_dst"} 6.0
faucet_config_table_names{dpid="0xeccd6df72de7",name="ipv6_fib"} 5.0
faucet_config_table_names{dpid="0xb827eb608918",name="flood"} 7.0
faucet_config_table_names{dpid="0xeccd6df72de7",name="eth_src"} 3.0
faucet_config_table_names{dpid="0xeccd6df72de7",name="port_acl"} 0.0
faucet_config_table_names{dpid="0xeccd6df2df80",name="ipv4_fib"} 4.0
faucet_config_table_names{dpid="0xeccd6df72de7",name="flood"} 7.0
faucet_config_table_names{dpid="0xb827eb608918",name="ipv4_fib"} 4.0
faucet_config_table_names{dpid="0xb827eb608918",name="vlan"} 1.0
faucet_config_table_names{dpid="0xeccd6df2df80",name="vlan"} 1.0
faucet_config_table_names{dpid="0xeccd6df2df80",name="ipv6_fib"} 5.0
faucet_config_dp_name{dpid="0xeccd6df72de7",name="windscale-faucet-1"} 260367057366503.0
faucet_config_dp_name{dpid="0xb827eb608918",name="link022-faucet-1"} 202481592207640.0
faucet_config_dp_name{dpid="0xeccd6df2df80",name="attq4600-faucet-1"} 260367057084288.0
faucet_config_reload_requests 0.0

**Comment by anarkiwi** [comment]
https://github.com/REANNZ/faucet/commit/b49014fd6688cb350e1080be458d30c82179ca04
